ouroboros-consensus-byron/test/Test/Consensus/Byron/Ledger/Golden.hs

```diff
@@ -529,6 +962,245 @@ test_golden_UPIState = goldenTestCBOR
   where
     exampleUPIState = CC.UPI.initialState CC.dummyConfig
 
+test_golden_HeaderHash :: Assertion
+test_golden_HeaderHash = goldenTestCBOR
+    encodeByronHeaderHash
+    exampleHeaderHash
+    [ TkBytes "y\USBV\225Lg\185\ETX\\;\128\160\130j\223q\157\&66\193\142\239\SYN\201\139\132\184\&3r=Q"
+    ]
+
+test_golden_GenTx :: Assertion
+test_golden_GenTx = goldenTestCBOR
+    encodeByronGenTx
+    getTx
+    [ TkListLen 2
+    , TkInt 0
+    , TkListLen 2
+    , TkListLen 3
+    , TkListBegin
+    , TkListLen 2
+    , TkInt 0
+    , TkTag 24
+    , TkBytes "\130X K\168\&9\196 \179\210\189C\149\&0\248\145\202\233\165\212\196\216\DC2\EOTF0\218\199.\142\tb\254\238\204\CAN/"
+    , TkBreak
+    , TkListBegin
+    , TkListLen 2
+    , TkListLen 2
+    , TkTag 24
+    , TkBytes "\131X\FS\170Sr\tZ\170h\r\EM\212\202Ii\131\161Ep\156;\225\139\rL\131\203{\220^\160\NUL"
+    , TkInt 853317774
+    , TkInt 47
+    , TkBreak
+    , TkMapLen 0
+    , TkListLen 1
+    , TkListLen 2
+    , TkInt 0
+    , TkTag 24
+    , TkBytes "\130X@Kmyw4lDSE5S4fSH6etNouiXezCyEjKc3tG4ja0kFjO8qzai26ZMPUEJfEy15ox5kX@h\138\173\133{\199\255\&0\252hb\218\ESC\226\129\244 \198Rq\183j\177\151\130\255@\226\149Z\248\136\EM\195\142\\y\DC3\143(\a:\186\225R\200\130%\139D \160\193\201\253\210l\152\129&\151\252>\NUL"
+    ]
+  where
+    getTx :: GenTx ByronBlock
+    getTx = ByronTx CC.exampleTxId (CC.annotateTxAux CC.exampleTxAux)
+
+test_golden_ApplyTxErr :: Assertion
+test_golden_ApplyTxErr = goldenTestCBOR
+    encodeByronApplyTxError
+    (CC.MempoolTxErr (CC.UTxOValidationTxValidationError (CC.TxValidationLovelaceError "a" (CC.LovelaceOverflow 0))))
```

Comment:
Make this a local `exampleApplyTxErr` (just like in the other tests) and make it fit on one line:

```haskell
  where
    exampleApplyTxErr =
        CC.MempoolTxErr
      . CC.UTxOValidationTxValidationError
      . CC.TxValidationLovelaceError "a"
      . CC.LovelaceOverflow 0
```